test/api/api_test.go

```diff
@@ -187,14 +195,20 @@ func RunApiServer(t *testing.T, binary string, storageApiHost string, repoPath s
 	envs.Set("ETCD_ENABLED", "false")
 
 	// Start API server
+	var stdout, stderr bytes.Buffer
 	cmd := exec.Command(binary, args...)
 	cmd.Env = envs.ToSlice()
-	cmd.Stdout = testhelper.VerboseStdout()
-	cmd.Stderr = testhelper.VerboseStderr()
+	cmd.Stdout = io.MultiWriter(&stdout, testhelper.VerboseStdout())
+	cmd.Stderr = io.MultiWriter(&stderr, testhelper.VerboseStderr())
```

Comment:
Ak API server spadne, tak sa teraz uz vypise aj `stdout` a `stderr` aby sa to dalo zdebugvat (bez nutnosti to spustat znova s `TEST_VERBOSE=true`).